apps/core/scenes/resultDetail/ResultDetail.js

```diff
@@ -94,6 +95,7 @@ class ResultDetail extends React.Component<Props, State> {
     return (
       <View style={styles.container}>
         <ScrollView>
+          <ResultDetailItineraryRenderer bookingToken={this.props.detailId} />
```

Comment:
this naming is confusing the one place we call this `detailId` an on another is `bookingToken`

Reply:
Good point. Changed to `bookingToken` (from schema) everywhere.